### ISSUE TITLE
PLEASE RE-CLONE / RE-FORK THIS REPO

### ISSUE THREAD

**Issue by Mytherin** [duckdb/duckdb-web#143]
In the context of #123 we have rewritten the history of this repository to remove (many) redundant large files. This has reduced the size of this repo from 1.3GB to 75MB. However, if you have an outstanding repository cloned or forked the old history will still be present there. It is recommended that you re-clone or re-fork this repository to prevent that from causing problems.

**Comment by Alex-Monahan** [comment]
As someone who had previously not re-forked, I figured I would leave a few extra details here!

The process I had to take was to delete my fork (double confirm, etc.), and then re-fork. I kept a local copy of the old fork on my system (to migrate the few files I had changed over to the new fork). I then renamed the local folder containing the old fork and cloned the new fork to a fresh folder.

Thanks for streamlining this Mark!
